src/applications/facility-locator/tests/reducers/facilities.unit.spec.js

```diff
@@ -46,6 +46,50 @@ describe('facilities reducer', () => {
     expect(state.pagination.currentPage).to.eql(1);
   });
 
+  it('should handle fetching state to build a search query object', () => {
+    const state = SearchResultReducer(INITIAL_STATE, {
```

Comment:
Would it also make sense to reuse the new query building helper in VAMap as part of this PR, since we are including changes related to the new helper here as well?